src/Servers/IIS/AspNetCoreModuleV2/RequestHandlerLib/requesthandler_config.h

```diff
@@ -255,6 +261,7 @@ class REQUESTHANDLER_CONFIG
     BOOL                   m_fWindowsAuthEnabled;
     BOOL                   m_fBasicAuthEnabled;
     BOOL                   m_fAnonymousAuthEnabled;
+    STRU                   m_fEnableOutOfProcessConsoleRedirection;
```

Comment:
Looking at the rest of requesthandler_config.h, and this seems to be the config value that is using a string to represent a bool instead of just using a bool directly.

It's unusual to have a boolean configuration value default to true. For things like AppContext switches, I know we have guidance is to *never* have a switch default to true. That's why [a lot of AppContext switches](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element#remarks) have "Dont", "Disable" or "No" in their names. Given the string acrobatics we're going through for the first time to default this bool config to true, it seems like the same convention has held for request handler config to this point, and I'm not sure we should break that expectation.

Reply:
We have a generic string bag for kvps. So it has to be a string initially s.t. we don't have a schema update. We then translate it to a bool.